PyDSS/dssInstance.py

```diff
@@ -337,6 +354,11 @@ def RunSimulation(self, project, scenario, file_prefix=''):
             step = 0
             while step < Steps:
                 self.RunStep(step)
+
+                if step == 0:
+                    size = make_human_readable_size(self.ResultContainer.max_num_bytes())
+                    self._Logger.info('Memory requirement estimation: %s, estimated based on first time step run.', size)
```

Comment:
This should say `Storage` instead of `Memory`.

Reply:
Changed.